feconf.py

```diff
@@ -469,6 +469,9 @@ def get_empty_ratings():
 # A sorted list of default categories.
 DEFAULT_CATEGORIES = sorted(CATEGORIES_TO_COLORS.keys())
 
+#Size of the gravatar in px assigned to the user after signup
+GRAVATAR_SIZE_PX = 50
```

Comment:
Added gravatar size in pixels in feconf.py ( I couldn't decide which size would be better so I took 50)